apps/jubmoji-quest/src/hooks/useFetchQuests.ts

```diff
@@ -0,0 +1,44 @@
+import { JubmojiQuest } from "@/types";
+import { useQuery } from "react-query";
+
+export const useFetchQuests = () => {
+  return useQuery(
+    ["quests"],
+    async (): Promise<JubmojiQuest[]> => {
+      const response = await fetch("/api/quests");
+
+      if (!response.ok) {
+        console.error("Could not fetch Jubmoji cards.");
+        return []; //
+      }
+
+      const quests = await response.json();
+
+      return quests;
+    },
+    {
+      refetchOnWindowFocus: false,
+    }
+  );
+};
+
+export const useFetchQuestById = (id: string | string[] | undefined) => {
```

Comment:
Same question here, is there a reason we allow fetching by `string[] | undefined`?